tests/cases/fourslash/quickInfoOnNarrowedType.ts

```diff
@@ -18,13 +18,26 @@
 ////   /*6*/s;
 ////}
 
+////class TestClass {

```

Comment:
Use generic _Foo/Bar/Baz_ for test class name instead?

Reply:
Good suggestion, why not?